frontend/packages/dev-console/src/components/topology/componentUtils.ts

```diff
@@ -226,8 +254,40 @@ const edgeDragSourceSpec = (
 
 const createConnectorCallback = (serviceBinding: boolean) => (
   source: Node,
-  target: Node,
+  target: Node | Graph,
+  dragEvent,
+  choice,
 ): any[] | null => {
+  const onKebabOptionClick = (option: KebabOption, sourceNode: Node) => {
+    if (option.callback) {
+      option.callback();
+    }
+    if (option.href) {
+      const sourceObj = _.get(sourceNode.getData(), ['resources', 'obj'], null);
+      const hrefWithContext = `${option.href}&contextSource=${referenceFor(sourceObj)}/${
+        sourceObj.metadata.name
```

Comment:
we should not be able to create knative resources using the drag connector, because knative revision can only be a source, it can never be a target. add `isKnativeDisabled` param here and remove it from other place.

Reply:
added `isKnativeDisabled` param

Reply:
We shouldn't need this param. The add forms should properly handle this case by identifying there is a `contextSource` present and therefore remove knative service option.